eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -332,17 +350,21 @@ jobs:
             echo "##vso[task.setvariable variable=additionalBuildArgs]--with-sdk /vmr/.dotnet"
           fi
 
-          docker run --rm -v "$(sourcesPath):/vmr" -w /vmr ${{ parameters.container }} ./prep-source-build.sh $customPrepArgs
+          cd $(sourcesPath)
+          ./prep-source-build.sh $customPrepArgs
         displayName: Prep the Build
 
     - script: |
         set -ex
         df -h
 
         customEnvVars=""
+        customPreBuildArgs=""
         customBuildArgs="--ci --clean-while-building --prepareMachine"
         if [[ '${{ parameters.runOnline }}' == 'True' ]]; then
           customBuildArgs="$customBuildArgs --online"
+        else
+          customPreBuildArgs="$customPreBuildArgs sudo unshare -n"
```

Comment:
Have you validated this required a as a privileged container and with sudo?

Reply:
Yes - my locally testing shows that running as a privileged container is still necessary, but running with sudo is not. Will update this PR later to reflect that once I verify via internal CI.

Reply:
Nevermind - apparently CI requires the use of sudo.